appveyor.bat

```diff
@@ -129,18 +130,22 @@ start /wait downloads\tcl.exe /extract:c:\ActiveTclTemp /exenoui /exenoupdates /
 for /d %%i in (c:\ActiveTclTemp\*) do move %%i %TCL_DIR%
 copy %TCL_DIR%\bin\%TCL_DLL% vim\src\
 
-:: Python 3.9
+:: Python 3
 call :downloadfile %PYTHON3_URL% downloads\python3.exe
 cmd /c start /wait downloads\python3.exe /quiet TargetDir=%PYTHON3_DIR%  Include_pip=0 Include_tcltk=0 Include_test=0 Include_tools=0 AssociateFiles=0 Shortcuts=0 Include_doc=0 Include_launcher=0 InstallLauncherAllUsers=0
 
 :: Ruby
+:: Download RubyInstaller binary
+call :downloadfile %RUBY_URL% downloads\ruby.7z
+7z x downloads\ruby.7z -oC:\ > nul || exit 1
+move C:\rubyinstaller-%RUBY_RELEASE%-%ARCH% %RUBY_DIR% > nul || exit 1
 :: RubyInstaller is built by MinGW, so we cannot use header files from it.
 :: Download the source files and generate config.h for MSVC.
 rem git clone https://github.com/ruby/ruby.git -b %RUBY_BRANCH% --depth 1 -q ../ruby
```

Comment:
can we remove that comment?

Reply:
L144 can be removed.
L142-L143 are still relevant.

Reply:
I keep this for now.

Reply:
sure thanks